suite-native/app/e2e/pageObjects/alertSheetActions.ts

```diff
@@ -0,0 +1,17 @@
+class AlertSheetActions {
+    async tapPrimaryButton() {
+        await waitFor(element(by.id('@alert-sheet/primary-button')))
+            .toBeVisible()
+            .withTimeout(10000);
+        await element(by.id('@alert-sheet/primary-button')).tap();
```

Comment:
I think it's generally advisable not to repeat the element locators and create a const for it.

Reply:
fixed